cmd/clusterctl/cmd/describe_cluster.go

```diff
@@ -161,20 +162,32 @@ func runDescribeCluster(name string) error {
 // printObjectTree prints the cluster status to stdout.
 func printObjectTree(tree *tree.ObjectTree) {
 	// Creates the output table
-	tbl := uitable.New()
-	tbl.Separator = "  "
-	tbl.AddRow("NAME", "READY", "SEVERITY", "REASON", "SINCE", "MESSAGE")
+	tbl := tablewriter.NewWriter(os.Stdout)
+	tbl.SetHeader([]string{"NAME", "READY", "SEVERITY", "REASON", "SINCE", "MESSAGE"})
+
+	tbl.SetAutoWrapText(false)
+	tbl.SetHeaderAlignment(tablewriter.ALIGN_LEFT)
+	tbl.SetAlignment(tablewriter.ALIGN_LEFT)
+
+	tbl.SetCenterSeparator("")
+	tbl.SetColumnSeparator("")
+	tbl.SetRowSeparator("")
+
+	tbl.SetHeaderLine(false)
+	tbl.SetBorder(false)
+	tbl.SetTablePadding(" ")
```

Comment:
This table padding should be 2 spaces and not one as referenced here(https://github.com/kubernetes-sigs/cluster-api/pull/5942/files#r800650647). Easy to miss :). 

This will align the the formatting exactly to what we had before.